### ISSUE TITLE
Elevated lipoprotein lipase level

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5949]
**Preferred term label:** Elevated lipoprotein lipase level

**Synonyms**

**Definition (free text, please give PubMed ID)** Increase in the level of lipoprotein lipase in the blood., PMID:12483461

**Parent term (use hpo.jax.org/app)** Abnormal enzyme/coenzyme activity HP:0012379

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
